infrastructure/database/templates/08_sql_functions.template.sql

```diff
@@ -50,5 +50,19 @@ BEGIN
 END
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION public.nowcast_input(_nowcast_date date, _type_id int)
```

Comment:
maybe call it `nowcast_inference_input`?

Reply:
renamed and added comment